aws/resource_aws_ebs_snapshot_copy_test.go

```diff
@@ -13,7 +12,9 @@ import (
 )
 
 func TestAccAWSEbsSnapshotCopy_basic(t *testing.T) {
-	var v ec2.Snapshot
+	var snapshot ec2.Snapshot
```

Comment:
To me: it would help if variable renaming was done as a separate commit so that the actual change can be validated independently of the renames. I'm now sure how the rest of the folks feel about it.

Reply:
Aye - good call. Will try to do that in the future.